inputs/worker_inner.json

```diff
@@ -0,0 +1,155 @@
+{
+  "client_version": "{{VERSION}}",
+  "prebuild_plugins": [
+    {
+      "name": "add_filesystem",
+      "args": {
+        "koji_hub": "{{KOJI_HUB}}",
+        "repos": []
+      }
+    },
+    {
+      "name": "pull_base_image",
+      "args": {
+        "parent_registry": "{{REGISTRY_URI}}",
+        "parent_registry_insecure": true
+      }
+    },
+    {
+      "args": {
+        "hub": "{{KOJI_HUB}}"
+      },
+      "name": "bump_release"
+    },
+    {
+      "args": {
+        "labels": "{{IMPLICIT_LABELS}}"
+      },
+      "name": "add_labels_in_dockerfile"
+    },
+    {
+      "name": "change_from_in_dockerfile"
+    },
+    {
+      "name": "add_help",
+      "required": false
+    },
+    {
+      "name": "add_dockerfile"
+    },
+    {
+      "args": {
+        "command": "{{SOURCES_COMMAND}}"
+      },
+      "name": "distgit_fetch_artefacts"
+    },
+    {
+      "args": {
+        "root": "{{KOJI_ROOT}}",
+        "target": "{{KOJI_TARGET}}",
+        "hub": "{{KOJI_HUB}}"
+      },
+      "name": "koji"
+    },
+    {
+      "args": {
+        "repourls": []
+      },
+      "name": "add_yum_repo_by_url"
+    },
+    {
+      "name": "inject_yum_repo"
+    },
+    {
+      "name": "distribution_scope"
+    }
+  ],
+  "buildstep_plugins": [
+    {
+      "name": "docker_api"
+    }
+  ],
```

Comment:
Removed `buildstep_plugins` as this feature is not in yet, and it's actually not needed since it should gracefully fallback to `docker_api` plugin once implemented.